incubator/hnc/pkg/stats/stats.go

```diff
@@ -42,7 +42,12 @@ var stats stat
 func StartHierConfigReconcile() {
 	stats.totalHierConfigReconciles.incr()
 	stats.curHierConfigReconciles.incr()
-	recordMetricsInt64(stats.totalHierConfigReconciles, hierConfigReconcileTotal)
+
+	m := int64(stats.totalHierConfigReconciles)
+	recordMetrics(m, hierConfigReconcileTotal)
+	// Only update the maxConcurrent value in StartReconcile() since it's impossible to
+	// get it from StopReconcile().
```

Comment:
Why's it impossible?

Reply:
Updated the comment with "since it's impossible to get the max from StopReconcile() when the reconcile number decrements"